rtlsdr/librtlsdr.py

```diff
@@ -58,76 +58,97 @@ def load_librtlsdr():
 
 # int rtlsdr_get_device_usb_strings(uint32_t index, char *manufact,
 #                                   char *product, char *serial)
+# * \return 0 on success
 f = librtlsdr.rtlsdr_get_device_usb_strings
 f.restype, f.argtypes = c_int, [c_uint,
                                 POINTER(c_ubyte),
                                 POINTER(c_ubyte),
                                 POINTER(c_ubyte)]
 
 # int rtlsdr_open(rtlsdr_dev_t **dev, uint32_t index);
+# FIXME: return value not documented
 f = librtlsdr.rtlsdr_open
 f.restype, f.argtypes = c_int, [POINTER(p_rtlsdr_dev), c_uint]
 
 # int rtlsdr_close(rtlsdr_dev_t *dev);
+# FIXME: return value not documented
 f = librtlsdr.rtlsdr_close
 f.restype, f.argtypes = c_int, [p_rtlsdr_dev]
 
 # /* configuration functions */
 
 # int rtlsdr_set_center_freq(rtlsdr_dev_t *dev, uint32_t freq);
+# * \return 0 on error, frequency in Hz otherwise
 f = librtlsdr.rtlsdr_set_center_freq
 f.restype, f.argtypes = c_int, [p_rtlsdr_dev, c_uint]
 
 # int rtlsdr_get_center_freq(rtlsdr_dev_t *dev);
+# * \return 0 on error, frequency in Hz otherwise
 f = librtlsdr.rtlsdr_get_center_freq
 f.restype, f.argtypes = c_uint, [p_rtlsdr_dev]
 
 # int rtlsdr_set_freq_correction(rtlsdr_dev_t *dev, int ppm);
+# * \return 0 on success
 f = librtlsdr.rtlsdr_set_freq_correction
 f.restype, f.argtypes = c_int, [p_rtlsdr_dev, c_int]
 
 # int rtlsdr_get_freq_correction(rtlsdr_dev_t *dev);
+# * \return correction value in parts per million (ppm)
+# NOTE: no return value indicated as an error
```

Comment:
The fact that there is no return value for this makes sense, at least I think it does.

If your `freq_correction` were set to a negative value, then there would be no way to indicate an error through a signed integer return.  A return value of "0" would be just as valid as <0 and >0 correct?